include/cartogram_info.hpp

```diff
@@ -48,7 +48,7 @@ class CartogramInfo
   [[nodiscard]] unsigned int n_insets() const;
 
 
-  void read_csv(const argparse::ArgumentParser &);
+  void read_csv(const Arguments &);
```

Comment:
Since `args` is already a member of `CartogramInfo`, the function signature can be `void read_csv()`;